diff --git a/build/config/siso/main.star b/build/config/siso/main.star
index 65e7e7d76fa3294dab0093090cdff242c030cef4..18c524e589ca8e0b2e0541ec8771230efa06aacd 100644
--- a/build/config/siso/main.star
+++ b/build/config/siso/main.star
@@ -23,6 +23,7 @@ load("./rust.star", "rust")
 load("./simple.star", "simple")
 load("./typescript_all.star", "typescript_all")
 load("./windows.star", chromium_windows = "chromium")
+load("./brave_siso_config.star", "brave_siso_config")
 
 def __setup_python(ctx, step_config):
     for rule in step_config["rules"]:
@@ -127,6 +128,7 @@ def init(ctx):
     handlers.update(reclient.handlers)
     handlers.update(typescript_all.handlers)
 
+    brave_siso_config.configure(ctx, step_config, filegroups, handlers)
     return module(
         "config",
         step_config = json.encode(step_config),
