diff --git a/chrome/browser/ui/web_applications/BUILD.gn b/chrome/browser/ui/web_applications/BUILD.gn
index 3eb307e96ac815061ea965fd36e4e454d355e568..9dd337fad24ac1d5acf7b630d22da16df2ca0323 100644
--- a/chrome/browser/ui/web_applications/BUILD.gn
+++ b/chrome/browser/ui/web_applications/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/ui/web_applications/sources.gni")
 import("//build/config/ui.gni")
 import("//extensions/buildflags/buildflags.gni")
 
@@ -101,6 +102,7 @@ source_set("launch_utils") {
 source_set("impl") {
   sources = []
   deps = [ "//chrome/browser/ui/browser_window:create_browser_window" ]
+  deps += brave_chrome_browser_ui_web_applications_deps
 
   if (!is_android) {
     sources += [ "web_app_dialog_utils.cc" ]
