diff --git a/chrome/browser/ui/tabs/public/tab_features.h b/chrome/browser/ui/tabs/public/tab_features.h
index e2f5067eced96784f07682588e94c2792fbb0e84..d10932908542574666d9ef6e35dd70d15c92c018 100644
--- a/chrome/browser/ui/tabs/public/tab_features.h
+++ b/chrome/browser/ui/tabs/public/tab_features.h
@@ -234,7 +234,7 @@ class TabInterface;
 class TabFeatures {
  public:
   TabFeatures();
-  ~TabFeatures();
+  virtual ~TabFeatures();
 
   TabFeatures(const TabFeatures&) = delete;
   TabFeatures& operator=(const TabFeatures&) = delete;
@@ -387,7 +387,7 @@ class TabFeatures {
   }
 
   // Called exactly once to initialize features.
-  void Init(TabInterface& tab, Profile* profile);
+  virtual void Init(TabInterface& tab, Profile* profile);
 
   static ui::UserDataFactoryWithOwner<TabInterface>&
   GetUserDataFactoryForTesting();
