diff --git a/chrome/browser/actor/execution_engine.cc b/chrome/browser/actor/execution_engine.cc
index d0eeb1908c25e98d1375fdfdf85935d6ee4bd054..4fc6402beb93ccc2d41e1f6c7db0a717300b391c 100644
--- a/chrome/browser/actor/execution_engine.cc
+++ b/chrome/browser/actor/execution_engine.cc
@@ -1294,6 +1294,16 @@ void ExecutionEngine::SafetyChecksForNextAction() {
   // TODO(mcnee): Add UMA for the outcomes.
   content::WebContents& web_contents = *(tab->GetContents());
   const GURL& url = web_contents.GetPrimaryMainFrame()->GetLastCommittedURL();
+  BRAVE_MAY_ACT_ON_URL_INTERNAL(
+      base::BindOnce([](MayActOnUrlBlockReason block_reason) {
+        return BlockReasonToResultCode(block_reason,
+                                       /*for_navigation=*/false);
+      })
+          .Then(base::BindOnce(&ExecutionEngine::DidFinishAsyncSafetyChecks,
+                               GetActionSequenceWeakPtr(),
+                               tab->GetContents()
+                                   ->GetPrimaryMainFrame()
+                                   ->GetLastCommittedOrigin())))
   auto event = GateableEvent::kPageAction;
   origin_gating_checker_.ComputeGatingDecision(
       std::make_unique<PageActionGatingContext>(web_contents.GetWeakPtr()),
@@ -1607,6 +1617,7 @@ const EnterprisePolicyChecker& ExecutionEngine::GetEnterprisePolicyChecker()
 void ExecutionEngine::IsAcceptableNavigationDestination(
     const GURL& url,
     DecisionCallbackWithReason callback) {
+  BRAVE_MAY_ACT_ON_URL_INTERNAL(std::move(callback))
   auto event = GateableEvent::kNavigationRequest;
   origin_gating_checker_.ComputeGatingDecision(
       std::make_unique<OriginGatingDecisionContext>(), event, /*source=*/GURL(),
