diff --git a/chrome/browser/ui/views/side_panel/BUILD.gn b/chrome/browser/ui/views/side_panel/BUILD.gn
index 1b9537ab3c006f747221b1f1082190b1f3b8bf27..67db0847ea255c916c73f9f902a7f133b2ff3fce 100644
--- a/chrome/browser/ui/views/side_panel/BUILD.gn
+++ b/chrome/browser/ui/views/side_panel/BUILD.gn
@@ -178,6 +178,7 @@ source_set("side_panel") {
     "//ui/webui",
   ]
 
+  import("//brave/browser/ui/views/side_panel/sources.gni") deps += brave_browser_ui_views_side_panel_deps public_deps += brave_browser_ui_views_side_panel_public_deps
   if (enable_extensions) {
     public_deps += [
       "//extensions/browser",
