diff --git a/components/metrics/BUILD.gn b/components/metrics/BUILD.gn
index 2612787d93cad1f8f25ebab5d65b06c9a7d847d0..53cd053f4187739dc8f7f0759ad753394d42645e 100644
--- a/components/metrics/BUILD.gn
+++ b/components/metrics/BUILD.gn
@@ -376,6 +376,7 @@ if (!is_cronet_build || is_ios) {
         "motherboard_metrics_provider.h",
       ]
     }
+    import("//brave/components/metrics/sources.gni") deps += brave_components_metrics_deps
   }
 
   static_library("test_support") {
