diff --git a/content/browser/web_contents/web_contents_impl.h b/content/browser/web_contents/web_contents_impl.h
index bb2f5d729673f74ae2a5b09f4820a6189b2a8130..419664793af664a406caf5dfbff8394061937037 100644
--- a/content/browser/web_contents/web_contents_impl.h
+++ b/content/browser/web_contents/web_contents_impl.h
@@ -1700,6 +1700,8 @@ class CONTENT_EXPORT WebContentsImpl
   // base::trace_event::TraceSessionObserver implementation:
   void OnStart(const perfetto::DataSourceBase::StartArgs&) override;
 
+  bool GetShouldDoLearningForTesting() override;
+
  private:
   using FrameTreeIterationCallback = base::FunctionRef<void(FrameTree&)>;
   using RenderViewHostIterationCallback =
