diff --git a/DEPS b/DEPS
index 96acce471557be4d6fe640d34d2756f13e663545..8616144a08fc10df2cd7cfdab2dc65a12db9454f 100644
--- a/DEPS
+++ b/DEPS
@@ -4503,7 +4503,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
