diff --git a/DEPS b/DEPS
index 3addee4bbdf5c093e3a254d8fe6246a8023b03c2..af46a59d2b70027157f0f2cbc9898664de5dd23c 100644
--- a/DEPS
+++ b/DEPS
@@ -4513,7 +4513,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
