diff --git a/net/BUILD.gn b/net/BUILD.gn
index f082a9f52d4a7ec89534c7b0c204f3df36e38e33..3eeabba388d4ad8f8373da91ab8d30edf9838128 100644
--- a/net/BUILD.gn
+++ b/net/BUILD.gn
@@ -1292,6 +1292,7 @@ component("net") {
       "reporting/reporting_uploader.h",
     ]
   }
+  sources += brave_net_sources import("//brave/components/brave_wallet/sources.gni") deps += brave_net_deps
 
   if (enable_device_bound_sessions) {
     sources += [
