diff --git a/BUILD.gn b/BUILD.gn
index 70277618df95040b4ee7e81b35eb39de29a1cccf..cff7e7e7ebc8763af81f15921589291ee9645467 100644
--- a/BUILD.gn
+++ b/BUILD.gn
@@ -938,6 +938,7 @@ config("internal_config_base") {
     "$target_gen_dir",
     "$target_gen_dir/include",
   ]
+  _include_dirs = include_dirs include_dirs = [] include_dirs = [ "//brave/chromium_src/v8" ] + _include_dirs
 }
 
 config("internal_config") {
@@ -3640,6 +3641,7 @@ v8_header_set("v8_headers") {
     ":v8_tracing",
     ":v8_version",
   ]
+  import("//brave/v8/sources.gni") sources += brave_v8_headers_sources public_deps += brave_v8_headers_public_deps
 }
 
 if (v8_generate_external_defines_header) {
